client-app/src/Bootstrap.ts

```diff
@@ -14,13 +14,17 @@ import {OauthService} from './core/svc/OauthService';
 import {TaskService} from './examples/todo/TaskService';
 
 declare module '@xh/hoist/core' {
+    // @ts-ignore - A 'duplicate declaration' that helps IntelliJ better recognize uses of app services from the `XH` singleton.
+    export const XH: XHApi;
+    // Tells typescript to *statically* assume that these services/fields exist on the `XH` singleton.
```

Comment:
I wonder if tweaking the comment a bit like this makes sense?

    // Merge interface with XHApi class to include injected services.

https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces

Reply:
I like this better, thanks!